feature-libs/checkout/base/public_api.ts

```diff
@@ -0,0 +1,5 @@
+/*
```

Comment:
not related, but reminder for us to include styles to each sub/multi entrypoints 

base, b2b, repl